backend/package.json

```diff
@@ -24,10 +24,12 @@
     "nodemon": "2.0.19",
     "pino-pretty": "8.1.0",
     "ts-node": "10.8.2",
-    "tsc-alias": "1.6.11"
+    "tsc-alias": "1.6.11",
+    "@types/bcrypt": "5.0.0"
```

Comment:
Please take a look at [this comment](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/15/files#r939562920) - seems like you've done the same fault